logic/transaction_pool.js

```diff
@@ -495,41 +495,48 @@ TransactionPool.prototype.reindexQueues = function() {
 TransactionPool.prototype.processBundled = function(cb) {
 	const bundled = self.getBundledTransactionList(true, self.bundleLimit);
 
-	async.eachSeries(
-		bundled,
-		(transaction, eachSeriesCb) => {
-			if (!transaction) {
-				return setImmediate(eachSeriesCb);
-			}
-			library.balancesSequence.add(balancesSequenceCb => {
-				__private.processVerifyTransaction(transaction, true, err => {
-					// Remove bundled transaction after asynchronous processVerifyTransaction to avoid race conditions
-					self.removeBundledTransaction(transaction.id);
-					// Delete bundled flag from transaction so it is qualified as "queued" in queueTransaction
-					delete transaction.bundled;
-
-					if (err) {
-						library.logger.debug(
-							`Failed to process / verify bundled transaction: ${
-								transaction.id
-							}`,
-							err
-						);
-						return setImmediate(balancesSequenceCb);
+	library.balancesSequence.add(
+		balancesSequenceCb => {
+			async.eachSeries(
+				bundled,
+				(transaction, eachSeriesCb) => {
```

Comment:
As we discussed earlier shouldn't we change that `eachSeriesCb` to `eachBundledTransacationCb`

Reply:
I am not sure about this approach. Since this will decrease the readability of the code. The problem occurs when we shadow variables, which can be avoided without making really lengthy variable names.